hibernate-core/src/main/java/org/hibernate/internal/CoreMessageLogger.java

```diff
@@ -1115,8 +1115,8 @@ void unableToInstantiateConfiguredSchemaNameResolver(
 	void unableToJoinTransaction(String transactionStrategy);
 
 	@LogMessage(level = INFO)
-	@Message(value = "Error performing load command : %s", id = 327)
-	void unableToLoadCommand(HibernateException e);
+	@Message(value = "Error performing load command :", id = 327)
```

Comment:
Let's remove the trailing ` :` as it makes no sense to have it anymore. Otherwise, looks good and it's something we sure want.

Reply:
Fixed

Reply:
Can you also fix it in the 5.3 PR? Thanks!

Reply:
Done